cmd/kubeadm/app/preflight/checks.go

```diff
@@ -213,11 +214,26 @@ func (hst HttpProxyCheck) Check() (warnings, errors []error) {
 	return nil, nil
 }
 
+type ImagePrefixCheck struct {
+	imagePrefix string
+}
+
+func (ipc ImagePrefixCheck) Check() (warnings, errors []error) {
+	if ipc.imagePrefix == kubeadmapiV1.DefaultImagePrefix {
+		return
+	}
+	warnings = []error{
+		fmt.Errorf("custom image prefix(%s) detected, please confirm whether you need to modify the kubelet `pod-infra-container-image` flag", ipc.imagePrefix),
```

Comment:
What does this mean? How/what should be modified here?  Without some sort of guide this is not actionable for users.